lib/logux/error_renderer.rb

```diff
@@ -16,7 +16,7 @@ def message
         build_message(exception, exception.message)
       when StandardError
         # some runtime error that should be fixed
-        ['error', 'Please look server logs for more information']
+        ['error', exception.backtrace]
```

Comment:
I suppose rendering full stack trace wouldn't be useful for non-test/development environment, but checking `Rails.env` here isn't flexible also

What do you think about introducing config options for this case:
```ruby
[
  'error',
  Logux.render_backtrace_on_error ? exception.backtrace : 'Please look server logs for more information'
]
```
which is not able by default.

and add https://github.com/wilddima/logux_rails/blob/master/lib/logux.rb#L51 the option here

Reply:
Done.